VargasMartinezJose/Dia3/HackerRank/JavaInheritance2.java

```diff
@@ -0,0 +1,17 @@
+public class JavaInheritance2 {
+
+    public static final Scanner scan = new Scanner(System.in);
+
+    public static void main(String[] args) {
+
+        int integer_value = scan.nextInt();
```

Comment:
Java prefers CamelCase to name variables

integerValue, doubleValue, stringValue